plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -488,6 +488,9 @@ public void testDate()
     @Test(dataProvider = "sessionZonesDataProvider")
     public void testMySqlDatetimeType(ZoneId sessionZone)
     {
+        // Skip unless https://github.com/trinodb/trino/pull/8060 is merged
+        skipTestUnless(false);
```

Comment:
Update the test so that it passes, instead.

This may look as an additional work, but even if this commit remained as part of https://github.com/trinodb/trino/pull/8060, we should do this anyway (we strive so that each commit should be a transition from a working state to a working state)